cpu/ppc64_cpu_test.py

```diff
@@ -37,14 +38,18 @@ def setUp(self):
         """
         Verifies if powerpc-utils is installed, and gets current SMT value.
         """
-        command = "uname -p"
-        if 'ppc' not in process.system_output(command, ignore_status=True):
+        dist = distro.detect()
+        if 'ppc' not in dist.arch:
```

Comment:
can it be like 

if 'ppc' not in distro.detect().arch

Reply:
sure